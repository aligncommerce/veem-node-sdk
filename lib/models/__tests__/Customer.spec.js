import Customer from 'models/Customer'

describe('Customer Model', () => {
  describe('response', () => {
    it('should match the snapshot', () => {
      expect(Customer.response.schema).toMatchSnapshot()
    })

    it('should define a valid schema', () => {
      expect(Customer.response.schema).to.be.validJsonSchema
    })
  })
})
