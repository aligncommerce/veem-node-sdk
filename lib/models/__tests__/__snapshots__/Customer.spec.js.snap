// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Customer Model response should match the snapshot 1`] = `
Object {
  "properties": Object {
    "email": Object {
      "type": "string",
    },
    "firstName": Object {
      "type": "string",
    },
    "id": Object {
      "type": "number",
    },
    "isContact": Object {
      "type": "boolean",
    },
    "isoCountryCode": Object {
      "type": "string",
    },
    "lastName": Object {
      "type": "string",
    },
    "name": Object {
      "type": "string",
    },
  },
  "title": "CustomerResponse",
  "type": "object",
}
`;
